tests/make_test_measurements.py

```diff
@@ -0,0 +1,87 @@
+"""Script to generate fake measurements-data for the telcell project."""
+
+import random
+import datetime
+from datetime import timedelta, timezone, datetime
+
+import numpy as np
+import pandas as pd
+
+df_list = []
+
+startdate = datetime(2023, 5, 17, 12, 00, 00, tzinfo=timezone.utc)
+
+grid_length = 36
+# Create equally spaced WGS-84 coordinates within the range of the original data
+lon_range = np.linspace(start=3.60848669870189, stop=6.97261243364462, num=grid_length)
+lat_range = np.linspace(start=50.7885407786764, stop=53.2155513728185, num=grid_length)
+
+for name in ["Bas", "Tim", "Stijn", "Daan", "Koen", "Martijn", "Pieter", "Maarten", "Henk", "Hans"]:
+    track = []
+    sensor = []
+    timestamp = []
+    cell_identifier = []
+    lon = []
+    lat = []
+    degrees = []
+    lon_idxs = []
+    lat_idxs = []
+    date = startdate
+    lon_idx, lat_idx = round(grid_length / 2), round(grid_length / 2)  # Starting position (center)
+
+    for day in range(5):
+        for measurement in range(20):
+            track.append(name)
+            sensor.append(name + "1")
+
+            date += timedelta(minutes=random.randint(1, 30))
+            timestamp.append(date.isoformat(" "))
+
+            cell_identifier.append("123-4-5678-9012")
+
+            # Randomly choose between going Up/Down, Left/Right on the grid
```

Comment:
You could choose here to give the combination dx,dy = (0,0) a much higher chance so that the tracks become more realistic